zoomx/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
     implementation 'com.android.support:recyclerview-v7:26.1.0'
 
     implementation "android.arch.persistence.room:runtime:1.0.0"
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
```

Comment:
why you added this

Reply:
Removed, dependency added by default when create activity.!